### PR TITLE
fix: add tests for batch ops & specifying null removes field in the update

### PR DESCRIPTION
**Pull request by Finistere**
fix: add tests for batch ops & specifying null removes field in the update

# Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: cli/changelog/0.27.0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### Features
+
+- Add create and update batch operations `postCreateMany` and `postUpdateMany`.
+
+### Bug fixes
+
+- For update mutation, like `postUpdate`, nullable fields can now be set to `null`. Previously, those update
+  would be ignored.
```
Comment (obmarg) [inline cli/changelog/0.27.0.md:8]:
```suggestion
- For update mutations, like `postUpdate`, nullable fields can now be set to `null`. Previously, those updates
  would be ignored.
```

**Review by obmarg — APPROVED** [review]
